sdk/src/env.rs

```diff
@@ -48,6 +48,10 @@ impl Env {
     pub fn del_contract_data<K: IntoTryFromVal>(&self, key: K) {
         internal::Env::del_contract_data(self, key.into_val(self));
     }
+
+    pub fn create_contract_using_parent_id(&self, contract: Object, salt: Object) {
```

Comment:
See the TODO comment above: this should be a typed interface.

Reply:
Should this instead just use two `Vec<u8>`? The host function expects both parameters to be of type `ScObject::Binary`, so why accept anything else?

Reply:
Are you saying that this interface should accept any one of the Object sub types?

Reply:
I'm saying that if the host function expects them to binary, then they should accept the SDK type `Binary` (see sdk/rs/binary.rs).

Reply:
Done. I left `create_contract` with `Object` parameters so I could use the test case in https://github.com/stellar/rs-stellar-contract-env/pull/144, but it should be possible to use `Binary` instead right?